### ISSUE TITLE
Pipeline automation?

### ISSUE THREAD

**Issue by simonrolph** [gjearevoll/Hotspots#2]
Hi there, Francesca Mancini sent a link to this repo and looks very interesting.

I was wondering if you're looking to implement any sort of automation to the pipeline? Eg. with the R package targets. There was a paper out recently which looked at the use of worklow management software (WMS) https://besjournals.onlinelibrary.wiley.com/doi/full/10.1111/2041-210X.14113 but there's loads of other tools out there (snakemake, airflow, galaxy etc.)

Basically, in our work we're also considering these tools and weighing up pros and cons of all the different tools out there for biodiversity analyses so would be interested if you're intending on automating and if you're planning on using any particular WMS?

**Comment by samaperrin** [comment]
Hi Simon, thanks for your interest! This project is in its infancy, but I'm trying to design it so that it can be automatically run from the command line, or even just automatically run monthly with the only thing we interact with being a run diagnostics report that lands in our emails. We haven't thought about any specific packages yet, but we'll give the paper and targets a look!
